bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/api/EnedisHttpApi.java

```diff
@@ -61,7 +62,7 @@ public class EnedisHttpApi {
     private static final String URL_ENEDIS_AUTHENTICATE = URL_APPS_LINCS + "/authenticate?target=" + URL_COMPTE_PART;
     private static final String USER_INFO_URL = URL_APPS_LINCS + "/userinfos";
     private static final String PRM_INFO_BASE_URL = URL_APPS_LINCS + "/mes-mesures/api/private/v1/personnes/";
-    private static final String PRM_INFO_URL = PRM_INFO_BASE_URL + "null/prms";
+    private static final String PRM_INFO_URL = URL_APPS_LINCS + "/mes-prms/api/private/v2/personnes/%s/prms";
     private static final String MEASURE_URL = PRM_INFO_BASE_URL
             + "%s/prms/%s/donnees-%s?dateDebut=%s&dateFin=%s&mesuretypecode=CONS";
```

Comment:
@clinique : is it normal that MEASURE_URL still uses the old v1 API while PRM_INFO_URL now uses the v2 API ?

Reply:
Yes, it is. The website itself uses a mix of v1 and v2. I thought the same.